### DISCUSSION TITLE
API and config file

### DISCUSSION THREAD

**Discussion by emiltin** [bluenviron/mediamtx#1203] [Questions] [answered]
Hi,
Looking at https://aler9.github.io/rtsp-simple-server I see it's possible to add/update paths, etc. but don't see a way to just reload the config file?

If I use the API to e.g. update a path, is the config file changed?

**Comment by aler9** [comment, marked as answer]
The configuration file doesn't store changes triggered through the API. If you need persistence, you can implement a wrapper that stores the configuration in a database, and fills the server configuration every time it starts.

[a comment by an automated account is not shown]
